### PR TITLE
Add chain 7078815900

### LINKED ISSUE #1739: Add Mekong Testnet
Mekong testnet is announced. We should add Sourcify support

https://blog.ethereum.org/2024/11/07/introducing-mekong-testnet

### PR DESCRIPTION
**Pull request by kuzdogan**
Fixes #1739 

Adds Mekong testnet: https://mekong.ethpandaops.io/

### CODE REVIEW

File: services/server/src/sourcify-chains-default.json [hunk 1]
```diff
@@ -126,6 +126,10 @@
       }
     }
   },
+  "7078815900": {
+    "sourcifyName": "Ethereum Mekong Testnet",
```
Comment (marcocastignoli) [inline services/server/src/sourcify-chains-default.json:130]: Maybe we can add the blockscout API:

https://explorer.mekong.ethpandaops.io/

File: services/server/src/sourcify-chains.ts [hunk 2]
```diff
@@ -15,6 +15,10 @@ import logger from "./common/logger";
 import fs from "fs";
 import path from "path";
 
+import dotenv from "dotenv";
+
+dotenv.config();
+
```
Comment (marcocastignoli) [inline services/server/src/sourcify-chains.ts:21]: Is this supposed to be included in this PR?

**Review by marcocastignoli — APPROVED** [review]
LGTM but if you want to add also the blockscout api support it's better 👍

File: services/server/src/sourcify-chains.ts (hunk 2 above)
Comment (kuzdogan) [reply services/server/src/sourcify-chains.ts:21]: Yes this is for https://github.com/ethereum/sourcify/pull/1747/commits/839b3be367f01389f7cb2ab9345908cccfa71f25 i.e. any script using `sourcify-chains.ts` (e.g. `deployContracts.ts`) needs this.

File: services/server/src/sourcify-chains-default.json (hunk 1 above)
Comment (kuzdogan) [reply services/server/src/sourcify-chains-default.json:130]: Nice! I wasn't aware of it, it wasn't listed in https://mekong.ethpandaops.io/

**Pull request merged**